src/Microsoft.AspNetCore.Mvc.Razor/Compilation/CompiledViewManifest.cs

```diff
@@ -13,13 +11,7 @@ public static class CompiledViewManfiest
     {
         public static readonly string PrecompiledViewsAssemblySuffix = ".PrecompiledViews";
```

Comment:
There's a couple of types referenced by precompilation But we need to switch to attributes first to get precompilation tests to work once we delete codes from it. This is first pass. Once this is in, we can delete a bunch of code from precompilation and then follow up with removing code from this repo.